spec/requests/api/v1/topics/show_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+describe "POST api/v1/topics", type: :request do
+
+  describe 'GET Index' do
+    context 'when the route is valid' do
+      before do
+        create_list(:topic, 5)
+        get api_v1_topics_path
+      end
+
+      it 'returns status 200' do
+        expect(response).to be_successful
+      end
+
+      it 'returns data attributes' do
```

Comment:
It feels to me that we are not testing this properly. 
How do we know that we are returning ALL topics and not only one?

Reply:
Ok